pkg/reconciler/addressableservice/addressableservice.go

```diff
@@ -18,35 +18,58 @@ package addressableservice
 
 import (
 	"context"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
 
+	corev1 "k8s.io/api/core/v1"
 	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	corev1listers "k8s.io/client-go/listers/core/v1"
 	"knative.dev/pkg/apis"
-	duckv1 "knative.dev/pkg/apis/duck/v1"
 	"knative.dev/pkg/logging"
 	"knative.dev/pkg/network"
 	"knative.dev/pkg/reconciler"
 	"knative.dev/pkg/tracker"
-	"knative.dev/sample-controller/pkg/apis/samples/v1alpha1"
+	v1alpha1 "knative.dev/sample-controller/pkg/apis/samples/v1alpha1"
+	"knative.dev/sample-controller/pkg/client/injection/reconciler/samples/v1alpha1/addressableservice"
 )
 
-// ReconcileKind implements Interface
-func (r *Reconciler) ReconcileKind(ctx context.Context, asvc *v1alpha1.AddressableService) reconciler.Event {
-	if asvc.GetDeletionTimestamp() != nil {
+// newReconciledNormal makes a new reconciler event with event type Normal, and
+// reason AddressableServiceReconciled.
+func newReconciledNormal(namespace, name string) reconciler.Event {
+	return reconciler.NewEvent(corev1.EventTypeNormal, "AddressableServiceReconciled", "AddressableService reconciled: \"%s/%s\"", namespace, name)
+}
+
+// Reconciler implements controller.Reconciler for AddressableService resources.
+type Reconciler struct {
```

Comment:
I'm trying to figure out where the type definition is on the left 🤔 

This shows as an addition, but I don't see a delete 😅

Reply:
"knative.dev/sample-controller/pkg/client/injection/reconciler/samples/v1alpha1/addressableservice"

Reply:
I updated the comment.